### PR TITLE
Fix unable to rollout ECS first taskset

### PR DESCRIPTION
**Pull request by khanhtc1202**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Follow PR #4600 

**Does this PR introduce a user-facing change?**:

- **How are users affected by this change**:
- **Is this breaking change**:
- **How to migrate (if breaking change)**:

### CODE REVIEW

File: pkg/app/piped/platformprovider/ecs/client.go [hunk 1]
```diff
@@ -275,8 +275,9 @@ func (c *client) GetServiceTaskSets(ctx context.Context, service types.Service)
 		activeTaskSetArns = append(activeTaskSetArns, *svc.TaskSets[i].TaskSetArn)
 	}
 
+	var taskSets []*types.TaskSet
 	if len(activeTaskSetArns) == 0 {
-		return nil, fmt.Errorf("failed to get task sets of service %s: services empty", *service.ServiceName)
+		return taskSets, nil
```
Comment (ffjlabo) [inline pkg/app/piped/platformprovider/ecs/client.go:280]: imo: just return `[]*types.TaskSet{}` and define `taskSets` variable to minify its scope. 👀

Comment (ffjlabo) [reply pkg/app/piped/platformprovider/ecs/client.go:280]: And it should be better to add comment for the reason.

Comment (khanhtc1202) [reply pkg/app/piped/platformprovider/ecs/client.go:280]: Okey, I don't have strong opinion on this 👍 Let's me adopt your suggestion

Comment (khanhtc1202) [reply pkg/app/piped/platformprovider/ecs/client.go]: @ffjlabo addressed by https://github.com/pipe-cd/pipecd/pull/4614/commits/8c6060cb8fe70574509af60da2ea5e2ebed40648

**Review by nghialv — APPROVED** [review]
🐬

**Pull request merged**